hack/lib/golang.sh

```diff
@@ -112,6 +112,10 @@ else
   # Which platforms we should compile test targets for. Not all client platforms need these tests
   readonly KUBE_TEST_PLATFORMS=(
     linux/amd64
+    linux/arm
+    linux/arm64
+    linux/s390x
+    linux/ppc64le
```

Comment:
will this cause any build performance regressions?

Reply:
no, this is for release (and the post-ci, independently running kube-cross job) code only. All other jobs etc perform the "fast path" and only build for amd64